lib/src/navigator/google_navigation_flutter_navigator.dart

```diff
@@ -50,9 +50,12 @@ class GoogleMapsNavigator {
   /// reporting abnormal SDK terminations such as the app crashes while the SDK is still running.
   ///
   static Future<void> initializeNavigationSession(
-      {bool abnormalTerminationReportingEnabled = true}) async {
+      {bool abnormalTerminationReportingEnabled = true,
+      TaskRemovedBehavior taskRemovedBehavior =
+          TaskRemovedBehavior.quitService}) async {
```

Comment:
Should this default to quitService, as this change will be breaking change?
@ArturoSalazarB16, what do you think?
At the moment by default the service keeps running on background even when application is killed (on Android)